WalletWasabi.Gui/Tabs/WalletManager/RecoverWallets/RecoverWalletViewModel.cs

```diff
@@ -49,6 +49,12 @@ public RecoverWalletViewModel(WalletManagerViewModel owner) : base("Recover Wall
 
 				string walletFilePath = WalletManager.WalletDirectories.GetWalletFilePaths(WalletName).walletFilePath;
 
+				bool isValidMinGapLimit = false;
+				if (int.TryParse(MinGapLimit, out int minGapLimit))
```

Comment:
Do not continue processing. This code is ignoring the fact that the entered value is invalid. It should break the circuit.

Reply:
I got it.